START.htm

```diff
@@ -177,17 +177,17 @@ <h2 id="update">Update board with latest software</h2>
     <img src="Docs/images/extract-latestimage.png" width="50%" /><br />
 
     <h3>Step #4: Install SD card programming utility</h3>
-    <p>Download and install <a target="_blank" href="https://launchpad.net/win32-image-writer/+download">Image Writer for Windows</a>. Be sure to download the binary distribution.</p>
-    <img src="Docs/images/download-win32diskimager.png" width="50%" /><br />
-    <img src="Docs/images/extract-win32diskimager.png" width="50%" /><br />
+    <p>Download and install <a target="_blank" href="https://etcher.io/">Etcher for Windows</a>.</p>
```

Comment:
Why restrict this to "Etcher for Windows"? We could just say "Download and install [Etcher](https://etcher.io/)"?